tests/test_with_credential.rs

```diff
@@ -81,6 +81,28 @@ async fn test_artists_albums() {
         .unwrap();
 }
 
+#[maybe_async::test(feature = "__sync", async(feature = "__async", tokio::test))]
+async fn test_artists_albums_with_multiple_album_types() {
+    let birdy_uri = ArtistId::from_uri("spotify:artist:2WX2uTcsvV5OnS0inACecP").unwrap();
+    let include_groups = [
+        AlbumType::Album,
+        AlbumType::Single,
+        AlbumType::Compilation,
+        AlbumType::AppearsOn,
+    ];
+    creds_client()
```

Comment:
In order to cover all cases, you should add a test case to test `artists_albums_with_zero_album_type`.